aten/src/ATen/native/cuda/Blas.cpp

```diff
@@ -713,4 +714,77 @@ Tensor _int_mm_cuda(const Tensor& self, const Tensor& mat2) {
   return _int_mm_out_cuda(self, mat2, result);
 }
 
+// Computes matrix multiply while applying scaling to input and output matrices and computes amax
+// Scales are only applicable when matrices are of Float8 type and assumbed to be equal to 1.0 by default
+// If output matrix type is 16 or 32-bit type, neither scale is applied nor amax is computed
```

Comment:
hmm, if the inputs are float8 and the output is float16/float32, don't we still need scales for the inputs?

Reply:
Ok, let me clarify that it only applies to output scale (i.e. input scales do apply, but output scale is no-op
Another thing probably worth documenting, that amax is computed before scaling